nemo_skills/dataset/mmmlu/prepare.py

```
@@ -0,0 +1,86 @@
+        **get_mcq_fields(entry),
+    }
+
+
+def main(args):
+    languages = [lang for lang in args.languages if lang != "EN-US"]
+    if args.include_english:
+        languages.append("EN-US")
+
+    invalid = set(languages) - set(SUPPORTED_LANGUAGES)
+    if invalid:
+        raise ValueError(f"Unsupported languages: {invalid}")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)